charts/prometheus-elasticsearch-exporter/values.yaml

```
@@ -94,6 +94,8 @@ commonLabels: {}
 
 affinity: {}
 
+topologySpreadConstraints: {}
```

`spec.topologySpreadConstraints` is an array:

```suggestion
topologySpreadConstraints: []
```